storage/rocksdb/ha_rockspart.cc

```diff
@@ -0,0 +1,118 @@
+/* Copyright (C) 2018 Percona
```

Comment:
Shouldn't this file be listed in CMakeLists.txt?

Reply:
It's listed in cmake-file. Please see storage/rocksdb/CMakeLists.txt.

Reply:
It's listed. Please see storage/rocksdb/CMakeLists.txt.